code/datums/components/tackle.dm

```diff
@@ -373,6 +380,9 @@
 	if(HAS_TRAIT(user, TRAIT_CLUMSY))
 		oopsie_mod += 6 //honk!
 
+	if(isflyperson(user))
+		oopsie_mod += 6 // flies don't take smacking into a window/wall easily
```

Comment:
Don't hardcode species checks. Make this a trait or something.

Reply:
Better yet, make it a var attached to physiology